docs/neurobagel_environment_variables.tsv

```diff
@@ -19,4 +19,6 @@ Environment variable	Set manually in .env?	Description	Default value if not set
 `NB_GRAPH_PORT`	No	Port number used by the _graph server container_	`7200` *	Docker, Python
 `NB_QUERY_TAG`	No	Docker image tag for the query tool	`latest`	Docker
 `NB_QUERY_PORT_HOST`	No	Port number used by the `query_tool` on the host machine	`3000`	Docker
-`NB_FEDERATE_REMOTE_PUBLIC_NODES`	Yes 	If "True", include public nodes in federation	`true`	Docker, Python
\ No newline at end of file
+`NB_FEDERATE_REMOTE_PUBLIC_NODES`	Yes 	If "True", include public nodes in federation	`true`	Docker, Python
+`NB_ENABLE_AUTH`	Yes	Whether to enable authentication for cohort queries. One of [true, false]	`true`	Docker, Python
+`NB_QUERY_CLIENT_ID`	Yes	OAuth client ID for the query tool. Required if NB_ENABLE_AUTH is set to true.	-	Docker, Python
```

Comment:
Like below, I think we should clarify here that these variables are not (yet) needed for prod deployments because auth is under active development still. Not sure how to best do that. Maybe you have a good idea?

Reply:
How about in parentheses at the start of the description `(EXPERIMENTAL)` / `(PREVIEW)` ? Or something similar

Reply:
Will use (Experimental, for dev deployments only) for now :)